randovania/games/super_metroid/layout/super_metroid_patch_configuration.py

```diff
@@ -0,0 +1,45 @@
+import dataclasses
+from typing import Iterator
+from enum import Enum
+
+from randovania.bitpacking.bitpacking import BitPackDataclass, BitPackEnum
+from randovania.bitpacking.json_dataclass import JsonDataclass
+
+
+class MusicMode(BitPackEnum, Enum):
+    VANILLA = "vanilla_music"
+    RANDOMIZED = "random_music"
+    OFF = "no_music"
+
+
+@dataclasses.dataclass(frozen=True)
+class SuperPatchConfiguration(BitPackDataclass, JsonDataclass):
+    music: MusicMode
```

Comment:
some of these really ought to be in the cosmetic patches instead of the configuration itself. I'd say you should move:
- music
- colorblind
- max ammo display
- aiming with any button
- removing the gameplay demo
- backup saves?
- better decompression?

over to cosmetic patches. basically, anything that isn't going to affect a race should be in cosmetic patches instead of in the preset.

Reply:
ofc, you'll need to add a test for the cosmetic patches UI once you do this as well